samples/sensor/vl53l0x/src/main.c

```diff
@@ -27,13 +27,13 @@ void main(void)
 			return;
 		}
 
-		ret = sensor_channel_get(dev, SENSOR_CHAN_PROX, &prox_value);
-		printk("prox is %d\n", prox_value.val1);
+		ret = sensor_channel_get(dev, SENSOR_CHAN_PROX, &value);
+		printk("prox is %d\n", value.val1);
 
 		ret = sensor_channel_get(dev,
 					 SENSOR_CHAN_DISTANCE,
-					 &prox_value);
-		printk("distance is %d\n", prox_value.val1);
+					 &value);
+		printk("distance is %.3fm\n", sensor_value_to_double(&value));
```

Comment:
This is not working with printk.
A working solution is to align on what is done for hts221 : 
    include  stdio.h instead of misc/printk.h
    use printf instead of printk

Reply:
Fixed